### DISCUSSION TITLE
realizes the equipment to communicate

### DISCUSSION THREAD

**Discussion by rui-ao** [strongswan/strongswan#1595] [Q&A] [answered]
I set up ipsec server on Ubuntu192.168.43.1, but will always fail to connect to the ipsec server

**Comment by tobiasbrunner** [comment, marked as answer]
What phone? Some clients (e.g. Apple iOS) exclude traffic to the local LAN explicitly from any VPN connections.

**Reply by rui-ao** [reply]
> What phone? Some clients (e.g. Apple iOS) exclude traffic to the local LAN explicitly from any VPN connections.
The phone is an android phone, I try oneplus 7 pro and a Huawei Pad, it is not working. I don't now what is wrang

**Reply by tobiasbrunner** [reply]
See #196.
